src/main.ts

```
@@ -1,8 +1,52 @@
+        transport: Transport.REDIS,
+        options: {
+          host: process.env.REDIS_HOST || 'localhost', // Use environment variable
+          port: parseInt(process.env.REDIS_PORT, 10) || 6379, // Use environment variable
+          // url: process.env.REDIS_URL || 'redis://localhost:6379', // Alternative using URL
+        },
+      },
+      { // gRPC
+        transport: Transport.GRPC,
+        options: {
+          package: process.env.GRPC_PACKAGE ? process.env.GRPC_PACKAGE.split(',') : ['hero'], // Use environment variable
+          protoPath: process.env.GRPC_PROTO_PATH ? join(__dirname, process.env.GRPC_PROTO_PATH) : join(__dirname, '_proto/hero.proto'), // Use environment variable
```

ESLint

[error] Replace `·?·join(__dirname,·process.env.GRPC_PROTO_PATH)` with `⏎············?·join(__dirname,·process.env.GRPC_PROTO_PATH)⏎···········`

(prettier/prettier)